source/extensions/filters/http/buffer/config.cc

```diff
@@ -19,9 +19,6 @@ namespace BufferFilter {
 Http::FilterFactoryCb BufferFilterFactory::createFilterFactoryFromProtoTyped(
     const envoy::config::filter::http::buffer::v2::Buffer& proto_config,
     const std::string& stats_prefix, Server::Configuration::FactoryContext& context) {
-  ASSERT(proto_config.has_max_request_bytes());
```

Comment:
Can you revert this and add required for the field in proto? I don't think it makes any sense for this to not be set? (Or perhaps `gt` requires it to be set? I'm not sure.)